### PR TITLE
Update avatars.

### LINKED ISSUE #1309: Add names and info to avatars
Add name and type to avatars on library details page based on this figma:
https://www.figma.com/design/o3pbrUqvILdvKw2crFGdC8/Untitled?node-id=0-1&m=dev&t=HaoUHhbIxULfV8Nv-1

Image

### PR DESCRIPTION
**Pull request by brianjp93**
- fixes #1309

### CODE REVIEW

**Comment by brianjp93** [comment]
image
image

3 column breakpoint
image
2
image
1
image

File: templates/partials/avatar.html [hunk 1]
```diff
@@ -1,31 +1,51 @@
...
-        <img src="{{ av_image_url }}"
-          alt="{{ av_alt }}"
-          class="
-            rounded-lg h-full w-full object-cover mx-auto
+  <div class="flex gap-x-2 items-center hover:bg-gray-100 dark:hover:bg-black rounded-lg">
+    <a {% if av_href and av_is_link %}href="{{ av_href }}"{% endif %}>
+      <div class="w-min text-center flex flex-col justify-center items-center">
+        <div
+          class="{{ av_size }} bg-gray-300 dark:bg-slate flex items-center justify-center relative rounded-lg"
+          title="{{ av_title }}{% if av_is_new %} - NEW!{% endif %}">
```
[the remark this answers is not in the document]
Comment (brianjp93) [reply templates/partials/avatar.html:9]: Probably not.  I'll remove it

**Comment by brianjp93** [comment]
@rbbeeston Do the contributors on the release page also need to use this "wide" avatar? It's still using the default square avatar.

**Comment by rbbeeston** [comment]
Yes, it should stay consistent on the release page, since it's displayed in the same way. I'll put it in a separate ticket, though

**Pull request merged**